certified-connectors/Delinea Secret Server/apiProperties.json

```diff
@@ -0,0 +1,57 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "baseurl": {
+        "type": "string",
+        "format": "hostname",
+        "uiDefinition": {
+          "displayName": "Base URL to Secret Server",
+          "description": "Enter base URL for Secret Server",
+          "tooltip": "URL",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
```

Comment:
@tylerezimmerman please check the curly braces here, I think the enclosing one for connectionParameters is missing.